src/providers/fault-summary/cat-d/fault-summary.cat-d.ts

```diff
@@ -10,7 +10,7 @@ import { getCompetencyFaults, getCompetencyComment } from '../../../shared/helpe
 import {
   manoeuvreCompetencyLabelsCatD,
   manoeuvreTypeLabelsCatD,
-} from '../../../shared/constants/competencies/CatD-manoeuvres';
+} from '../../../shared/constants/competencies/catd-manoeuvres';
```

Comment:
The typical naming convention would be cat-d-manoeuvres

Reply:
'CatD' appears to be a mistake introduced a couple of weeks ago.  I changed this to 'catd' following the existing convention of the other files in that directory

Reply:
You are right about the existing files int he directory, a shame because we use the cat-X convention everywhere else!